app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,65 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      def show
+        volunteer = Volunteer.find(params[:id])
+        render json: { status: 'SUCCESS', message: 'Loaded volunteer', data: volunteer }, status: :ok
+      end
+
+      def create
+        volunteer = Volunteer.new(volunteer_params)
+
+        if volunteer.save
+          render json: { status: 'SUCCESS', message: 'Created volunteer', data: volunteer }, status: :ok
+        else
+          render json: { status: 'ERROR', message: 'Volunteer not saved', data: volunteer.errors }, status: :unprocessable_entity
+        end
+      end
+
+      def update
+        volunteer = Volunteer.find(params[:id])
+
+        if volunteer.update(volunteer_params)
+          render json: { status: 'SUCCESS', message: 'Updated volunteer', data: volunteer }, status: :ok
+        else
+          render json: { status: 'ERROR', message: 'Volunteer not updated', data: volunteer.errors }, status: :unprocessable_entity
+        end
+      end
+
+      def destroy
+        volunteer = Volunteer.find(params[:id])
+        volunteer.destroy
+
+        render json: { status: 'SUCCESS', message: 'Deleted volunteer', data: volunteer }, status: :ok
+      end
+
+      def get
+        volunteers_json = fetchVolunteerJson;
+
+        respond_to do |format|
+            format.json do
+                render(json: volunteers_json)
+            end
+        end
```

Comment:
There is a custom json response module located in `/app/controllers/concerns` which is a helper for rendering son. so in this case respond_to and the render method are redundant. This can simply read `json_response(volunteers_json)`

Reply:
Done